AutomationHelpers/src/UserCodeCollections/StringLibrary.cs

```diff
@@ -24,5 +24,33 @@ public static string ConcatStrings(string value1, string value2)
         {
             return String.Concat(value1, value2);
         }
+
+        /// <summary>
+        /// Creates and returns a random string. If a string shorter than 35 characters is used it may not be unique.
+        /// </summary>
+        /// <param name="length">Expected length of the return value (default length is 35)</param>
+        /// <returns>Random string</returns>
+        [UserCodeMethod]
+        public static string GetRandomString(string length)

```

Comment:
You might want to add a default parameter (= "35") like for the GetDateTimeAsString method

Reply:
fixed